src/Glpi/Form/Destination/CommonITILField/RequestTypeField.php

```diff
@@ -0,0 +1,285 @@
+                data-glpi-destination-parent="{{ main_config_field.input_name }}"
+                data-glpi-destination-parent-value="{{ CONFIG_SPECIFIC_ANSWER }}"
+            >
+                {{ fields.dropdownArrayField(
+                    specific_answer_extra_field.input_name,
+                    specific_answer_extra_field.value,
+                    specific_answer_extra_field.possible_values,
+                    "",
+                    options|merge({
+                        no_label: true,
+                        display_emptychoice: true,
+                        emptylabel: specific_answer_extra_field.empty_label,
+                    })
+                ) }}
+            </div>
+TWIG;
+
+        $twig = TemplateRenderer::getInstance();
+        return $twig->renderFromStringTemplate($template, $parameters);
+    }
+
+    #[Override]
+    public function applyConfiguratedValueToInputUsingAnswers(
+        mixed $configurated_value,
+        array $input,
+        AnswersSet $answers_set
+    ): array {
+        if (is_null($configurated_value)) {
+            return $input;
+        }
+
+        switch ($configurated_value['value']) {
+            default:
+            case self::CONFIG_FROM_TEMPLATE:
+                // Let the template apply its default value by itself.
+                $value = null;
+                break;
+
+            case self::CONFIG_SPECIFIC_VALUE:
+                $value = $this->getConfiguratedSpecificValue($configurated_value);
+                break;
+
+            case self::CONFIG_SPECIFIC_ANSWER:
+                $question_id = $this->getConfiguratedQuestionId($configurated_value);
+                if ($question_id === null) {
+                    $value = null;
+                    break;
+                }
+
+                $answer = $answers_set->getAnswerByQuestionId($question_id);
+                if ($answer === null) {
+                    $value = null;
+                    break;
+                }
+
+                $value = $answer->getRawAnswer();
+                break;
+
+            case self::CONFIG_LAST_VALID_ANSWER:
+                $valid_answers = $answers_set->getAnswersByType(
+                    QuestionTypeRequestType::class
+                );
+
+                if (count($valid_answers) == 0) {
+                    $value = null;
+                    break;
+                }
+
+                $answer = end($valid_answers);
+                $value = $answer->getRawAnswer();
+                break;
+        }
+
+        // Do not edit input if invalid value was found
+        $valid_values = [Ticket::INCIDENT_TYPE, Ticket::DEMAND_TYPE];
+        if (!array_search($value, $valid_values)) {
+            return $input;
+        }
+
+        // Apply value
+        $input['type'] = $value;
+        return $input;
+    }
+
+    #[Override]
+    public function getDefaultValue(Form $form): mixed
+    {
+        return ['value' => self::CONFIG_LAST_VALID_ANSWER];
```

Comment:
Not really, with `CONFIG_LAST_VALID_ANSWER` it will automatically take into account any request type question added into the form without the need for the administrator to think about it.

Our goal for this section is that 95% of users never need to changes these values.

Reply:
> Not really, with `CONFIG_LAST_VALID_ANSWER` it will automatically take into account any request type question added into the form without the need for the administrator to think about it.
> 
> Our goal for all configuration fields in this section is that 95% of users never need to changes these values.

Then maybe the default configuration should be something like "AUTO". The "AUTO" value could be used to get the answer for the latest `RequestType` question, if one exists, and fallback to the value defined in the corresponding ticket template.